docs/Release_Guide/metviewer_development.rst

```diff
@@ -7,5 +7,13 @@ Create a new vX.Y.Z-betaN or vX.Y.Z-rcN development release from the develop bra
 
 .. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
-.. include:: release_steps/checkout_main_branch.rst
-.. include:: release_steps/create_release_feature_branch.rst
+.. include:: release_steps/checkout_develop_branch.rst
+.. include:: release_steps/metviewer/update_version.rst
```

Comment:
Above this line ".. include:: release_steps/metviewer/update_version.rst" should be .. include:: release_steps/create_release_feature_branch.rst
(Purpose is to update the version number and add release notes and have those reviewed in a pull request and is referred to in release_steps/merge_release_issue.rst which is included already)